.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: false
+language: python
+install: pip install tox
+matrix:
+  include:
+    - python: "2.6"
+      env: TOX_ENV=py26
+    - python: "2.7"
+      env: TOX_ENV=py27
+    - python: "3.6"
+      env: TOX_ENV=py36
+    - python: "2.7"
+      env: TOX_ENV=static
+    # alt-src not available on PyPI
```

Comment:
I think this should include tox env cov-travis too for coverage report on coveralls while setting up the CI.